test/spec/airbyte-local_spec.sh

```diff
@@ -98,7 +98,7 @@ Describe 'building source catalog'
                 --src.feed_cfg.feed_name 'jira-feed' \
                 --src.feed_cfg.feed_path 'tms/jira-feed' \
                 --debug
-        The output should include 'Using source configured catalog: {"streams":[{"stream":{"name":"faros_feed","json_schema":{}},"sync_mode":"incremental","destination_sync_mode":"append"}]}'
+        The output should include 'Using source configured catalog: {"streams":[{"stream":{"name":"faros_feed","supported_sync_modes":["full_refresh","incremental"],"json_schema":{}},"sync_mode":"incremental","destination_sync_mode":"append"}]}'
```

Comment:
Why were the tests that didn't use `--src-catalog-overrides` not failing before?